module/localisation/RobotLocalisation/src/RobotLocalisation.cpp

```diff
@@ -190,35 +194,19 @@ namespace module::localisation {
             last_time_update_time = NUClear::clock::now();
         });
 
-        on<Trigger<WalkCommand>>().then([this](const WalkCommand& wc) {
-            if (!walk_engine_enabled) {
-                const auto current_time = NUClear::clock::now();
-                last_time_update_time   = current_time;
-                walk_engine_enabled     = true;
-            }
-            walk_command = wc.command;
-        });
-
-        on<Trigger<EnableWalkEngineCommand>>().then([this]() {
-            walk_engine_enabled     = true;
-            const auto current_time = NUClear::clock::now();
-            last_time_update_time   = current_time;
-        });
-
-        on<Trigger<DisableWalkEngineCommand>>().then([this]() {
-            walk_command        = Eigen::Vector3d::Zero();
-            walk_engine_enabled = false;
-        });
-
-        on<Trigger<StopCommand>>().then([this]() {
-            walk_command        = Eigen::Vector3d::Zero();
-            walk_engine_enabled = false;
-        });
-
         on<Trigger<ExecuteGetup>>().then([this]() { falling = true; });
 
         on<Trigger<KillGetup>>().then([this]() { falling = false; });
 
+        on<Trigger<Stability>>().then([this](const Stability& stability) {
+            if (stability == Stability::FALLEN) {
```

Comment:
is this working alright when kicking? I was having major problems with the localisation completely going off when kicking and so I added to this if stability::Standing for when the robot is kicking (hacky)

Reply:
Nah, kicking really messes it up, not sure why, adding something to stop localisation while kicking would be good for now.